config/swagger.yaml

```diff
@@ -240,7 +240,7 @@ definitions:
         format: int64
       nonce:
         type: integer
-        format: int64
+        format: uint64
```

Comment:
I see this definition brings the HTTP API spec (swagger.yaml) in line with the documentation of the nonce in the header as `u64` in https://github.com/aeternity/epoch/wiki/Block-Header - thanks.